src/MagnumPlugins/DrFlacAudioImporter/Test/DrFlacImporterTest.cpp

```diff
@@ -42,30 +42,83 @@ class DrFlacImporterTest: public TestSuite::Tester {
 
         void mono8();
         void mono16();
+        void mono24();
+
+        void stereo8();
         void stereo16();
+        void stereo24();
+
+        void quad16();
+        void quad24();
+
+        void surround51Channel16();
+        void surround51Channel24();
 };
 
 DrFlacImporterTest::DrFlacImporterTest() {
     addTests({
               &DrFlacImporterTest::mono8,
               &DrFlacImporterTest::mono16,
-              &DrFlacImporterTest::stereo16});
+//              &DrFlacImporterTest::mono24,
+
+              &DrFlacImporterTest::stereo8,
+              &DrFlacImporterTest::stereo16,
+//              &DrFlacImporterTest::stereo24,
+
+              &DrFlacImporterTest::quad16,
+//              &DrFlacImporterTest::quad24,
+
+              &DrFlacImporterTest::surround51Channel16
+//              &DrFlacImporterTest::surround51Channel24
```

Comment:
Why are these commented out?

In case the tests should pass but they don't (due to a limitation in the dr_flac implementation or due to a limitation in the plugin), then there should be [CORRADE_EXPECT_FAIL()](http://mosra.cz/blog/corrade-doc/Tester_8h.html#afe4fc83d4e99d4e56d989a6eb005f7c5) used. In case the test won't *ever* pass (because there's no 24bit format in OpenAL, e.g.), then they shouldn't be there at all.

Since there are no 24bit test files present, I assume the latter :)